src/custom/pages/Swap/index.tsx

```diff
@@ -163,7 +219,8 @@ function FeeGreaterMessage({ fee }: FeeGreaterMessageProp) {
         </MouseoverTooltipContent>
       </RowFixed>
       <TYPE.black fontSize={14} color={theme.text1}>
-        {formatSmart(fee, SHORT_PRECISION)} {fee.currency.symbol}
+        {formatSmart(fee, SHORT_PRECISION)} {fee.currency.symbol}{' '}
```

Comment:
PRECISION = 4 here. up for discussion. looks better smaller decimals as there's already lots of data in this line

Reply:
For the fee I think it's best to use the same as other amounts.
What I agree that should be changed is the USD estimations and percentages, to use only 2 decimals.
But that's not in the scope of this PR